wdl-analysis/src/graph.rs

```diff
@@ -105,6 +122,8 @@ impl Analysis {
 pub struct DocumentGraphNode {
     /// The URI of the document.
     uri: Arc<Url>,
+    /// The current incremental change to the document.
+    change: Option<IncrementalChange>,
     /// The parse state of the document.
     parse_state: ParseState,
     /// The analysis of the document.
```

Comment:
```suggestion
    /// The analysis of the document.
    ///
    /// If `None`, an analysis does not exist for the current state of the node.
```

Reply:
This is not related to the PR changes, but I caught it when I suggested the above change.